recipes/loghub/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "loghub" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "79c4867319f61257e7a643494910d35891fb08791951656731f315c73924b289" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - loghub = loghub.main:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - loghub
```

Comment:
This test is not that important because this is not a library. I think it's better

    commads:
        - loghub --help